### ISSUE TITLE
add word list widget

### ISSUE THREAD

**Issue by jtauber** [scaife-viewer/readhomer#15]
the vocab.perseus.org API should work so we should be able to port over the word list widget.

**Comment by jacobwegner** [comment]
removed the `backend-api` label as it seems like we're porting over a widget and consuming directly from `vocab.perseus.org`

**Comment by jtauber** [comment]
Yeah, it should work with the existing API so we should be good there.
